.sbtopts

```diff
@@ -1,5 +1,5 @@
--J-Xmx2g
--J-Xms1g
+-J-Xmx2G
+-J-Xms1G
 -J-Xss2M
 -Djava.awt.headless=true
--Djava.library.path=/usr/local/lib
\ No newline at end of file
+-Djava.library.path=/usr/local/lib
```

Comment:
This is also defined in the `build.sbt` file. Wondering where we'd like to keep this.

Reply:
I think it makes sense only to have this setting in the `build.sbt` file. And to add into `Environment.scala` smth like:

```scala
lazy val javaGdalDir = Properties.envOrElse("JAVA_GDAL_DIR", "/usr/local/lib")
```

To make it a bit easier for users to configure.